src/renderer/components/dock/dock.scss

```diff
@@ -76,4 +80,47 @@
       bottom: 0;
     }
   }
+
+  .tabs-control {
+    overflow: auto;
+    position: relative;
+
+    &::-webkit-scrollbar {
+      visibility: hidden;
+      height: 4px;
+      background:transparent;
+      position: absolute;
+      bottom: 0;
+    }
```

Comment:
Maybe we should use `@include hidden-scrollbar` from common `mixins.scss`?

Reply:
Tried importing hidden-scrollbar but unfortunately, that hides scroll bars completely. We need custom implementation here.

Reply:
That was the point (to hide it completely) imho.. Scrolling still should work + if we indicating it with arrows on left/right sides when available.

Reply:
My idea was also to show it when a user hovers over the tabs, so they can scroll quickly by grabbing the handle, the same way it's done in VSCode.